.github/workflows/Default.yml

```diff
@@ -0,0 +1,38 @@
+name: Log4JNA Default Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: ["8", "11", "17"]
+        distribution: ["temurin", "microsoft"]
+        exclude:
+          - java: "8"
+            distribution: "microsoft"

```

Comment:
There's a somewhat cleaner way of doing this, see https://github.com/slack-ruby/slack-ruby-bot-server/blob/master/.github/workflows/test-mongodb.yml#L10 for an example.

Reply:
resolved in #54